gpbft/participant.go

```diff
@@ -67,15 +67,12 @@ func NewParticipant(host Host, o ...Option) (*Participant, error) {
 	}, nil
 }
 
-// Fetches the preferred EC chain for the instance and begins the GPBFT protocol.
-func (p *Participant) Start() (err error) {
-	defer func() {
-		if r := recover(); r != nil {
-			err = &PanicError{Err: r}
-		}
-	}()
-
-	return p.beginInstance()
+// Start uses SkipToInstance under the hood to trigger
+// the start of the first instance.
+// This way we handle in an homogeneous way the start of
+// new instances
+func (p *Participant) Start() {
```

Comment:
Restored. Would you mind elaborating a bit on why this is needed? I think I may have misunderstood the purpose of that code. I was assuming it would be handled by the alarm handler. Thanks!